src/import/trace-event.ts

```diff
@@ -641,6 +686,31 @@ function isTraceEventList(maybeEventList: any): maybeEventList is TraceEvent[] {
   return true
 }
 
+function isHermesTraceEvent(traceEventArgs: any): traceEventArgs is HermesTraceEventArgs {
+  const requiredProperties: Array<keyof HermesTraceEventArgs> = [
+    'line',
+    'column',
+    'name',
+    'category',
+    'url',
+    'params',
+    'allocatedCategory',
+    'allocatedName',
+  ]
```

Comment:
Browsers probably optimize this anyway, but just for sanity, let's move this array outside of the call-stack. This seems like it would fit well as a constant directly below `HermesTraceEventArgs` as `heremesTraceEventRequiredProperties`